presto-hive/src/main/java/com/facebook/presto/hive/ColumnarBinaryHiveRecordCursor.java

```diff
@@ -636,6 +641,9 @@ private void parseColumn(int column)
         else if (BIGINT.equals(type)) {
             parseLongColumn(column);
         }
+        else if (INTEGER.equals(type)) {
+            parseLongColumn(column);
```

Comment:
According to the spec it should be: https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Types#LanguageManualTypes-IntegralTypes(TINYINT,SMALLINT,INT,BIGINT)

I'm concerned about the performance penalty of a check (or would branch prediction make this go away?).